### PR TITLE
Add an endpoint to list files owned by an assetstore.

### PR DESCRIPTION
**Pull request by manthey**
This resolves issue #927.

### CODE REVIEW

File: girder/api/v1/assetstore.py [hunk 1]
```diff
@@ -260,3 +261,18 @@ def updateAssetstore(self, assetstore, params):
...
+    @loadmodel(model='assetstore')
+    @describeRoute(
+        Description('Get a list of files controlled by an assetstore.')
+        .param('id', 'The assetstore ID.', paramType='path')
+        .pagingParams(defaultSort='_id')
+        .errorResponse()
+        .errorResponse('You are not an administrator.', 403)
+    )
+    def getAssetstoreFiles(self, assetstore, params):
+        limit, offset, sort = self.getPagingParameters(params, 'lowerName')
```
Comment (mgrauer) [inline girder/api/v1/assetstore.py]: why is there a different sort order within the method `'lowerName'` compared to the `@describeRoute`, `'_id'`?

Comment (manthey) [inline girder/api/v1/assetstore.py]: Copy and paste stupidity...  I meant for it to be by _id.

**Comment by mgrauer** [comment]
@manthey 

Could I prevail upon you to sneak in a similar fix in the [folder find endpoint](https://github.com/girder/girder/blob/master/girder/api/v1/folder.py#L62)?

It should be `'lowerName'`.

**Comment by mgrauer** [comment]
Thanks for the fix @manthey.  LGTM.

You squashed this bug 2 days shy of its first birthday.

🍰

**Pull request merged**